### DISCUSSION TITLE
Binding not found for drawble

### DISCUSSION THREAD

**Discussion by namnguyen279** [dlemstra/magick-wasm#117] [General]
### magick-wasm version

0.0.24

### Description

I created some of drawable classes like point, line. It seem the bind available in the API .. but calling it throw error No Binding found 

### Steps to Reproduce

1. Copy DrawbleText class to new file, rename DrawablePoint, change x, y according to export in api native
2. Export * from index.ts
3. add method to into drawing-wand to call API

### Images

![Screenshot 2023-10-25 at 23 26 43](https://github.com/dlemstra/magick-wasm/assets/8275961/d5db69be-d49c-4208-ad3f-8e7837ed9fd9)
![Screenshot 2023-10-25 at 23 26 48](https://github.com/dlemstra/magick-wasm/assets/8275961/4c7b1cce-12f3-4e8b-8dab-2c48a264f662)
![Screenshot 2023-10-25 at 23 27 02](https://github.com/dlemstra/magick-wasm/assets/8275961/2ae16b45-7860-4fb9-94ac-aec4579bb6eb)
![Screenshot 2023-10-25 at 23 28 58](https://github.com/dlemstra/magick-wasm/assets/8275961/967618f0-188c-49fa-bcf2-ad4cb0d2d99f)

**Comment by dlemstra** [comment]
I really don't understand what you are trying to do?

**Comment by namnguyen279** [comment]
Actually, i tried to draw stroke and line, but i tried to interface a simple binding but it doesnt work, do you mind show me how ?

**Comment by dlemstra** [comment]
It is still unclear for me what you did exactly and caused that error message. I just made this commit that probably explains what you missed: https://github.com/dlemstra/magick-wasm/commit/82851f19e8cc614772775942e9a903b43f142f79.

p.s. Next time please use a discussion instead of an issue when you have questions about this project.
